portal-impl/src/com/liferay/portal/security/permission/ResourceActionsImpl.java

```diff
@@ -996,7 +1003,8 @@ private String _normalizePortletName(
 
 	private void _read(
 			Portlet portlet, String servletContextName, ClassLoader classLoader,
-			String source, Set<String> portletNames)
+			String source, Set<String> resourceNames,
+			boolean readPortletResource)
```

Comment:
@dantewang @yuhai 
I do not like the new add parameter `readPortletResource`, could you please try to find a better way to do this?

Thanks.
Tina.